adala/runtimes/base.py

```diff
@@ -191,6 +216,11 @@ def record_to_batch(
             response_model=response_model,
         )
 
+    def get_cost_estimate(
```

Comment:
we can use @abc.abstractmethod

Reply:
I didn't want to force every runtime to define this, esp since I'm not sure how do-able it is for images yet